### DISCUSSION TITLE
Uses for binary variables and panel data?

### DISCUSSION THREAD

**Discussion by ghost** [mmeierer/REndo#81] [General]
Hi all. Thanks for the excellent work on REndo. I'm really enjoying the package so far!

My two questions are:
1.  Can these approaches be used for panel data or only cross-sectional?
2.  Are any of the approaches suitable for binary variables? Obviously linear models are commonly used in economics for these cases (as marginal effects are often similar).

Thanks again!

**Comment by mmeierer** [comment]
Thanks for the questions. I see that these are surely interesting to many.

@ 1: The implementations provided have been proposed for cross-sectional data with continuous DVs. See the respective help files where we cite which exact papers we refer to. I would suggest having a look at the literature and contacting the original authors to assess the usage for panel data.

@ 2: The same suggestion applies to your second question. The Copula approach, for example, can be technically applied to binary data but may require some modifications. See the original paper. 

Please feel free to make a pull request, should you come up with some code that extends the current functionalities of REndo.
